src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/PhrasalTemplate/PhrasalTemplatePhrase/PhrasalTemplatePhrase.tsx

```diff
@@ -105,6 +117,16 @@ export const PhrasalTemplatePhrase = ({
     swap(source.index, destination.index);
   };
 
+  const validatePhraseField = async (field: Record<'id', string>): Promise<boolean> => {
+    try {
+      await PhrasalTemplateResponseValuePhraseSchema.validate(field);
+
+      return false;
+    } catch (error) {
+      return true;
+    }
+  };
```

Comment:
The logic here seems to be backwards. A function that provides a validation result should be `true` if it's valid, `false` if invalid. I'll make a tweak to this to make the logic a little more intuitive.